src/main/java/eu/bbmri/eric/csit/service/negotiator/mappers/NegotiationModelMapper.java

```diff
@@ -81,11 +81,13 @@ void addMappings() {
 
   private Set<PersonRoleDTO> personsRoleConverter(Set<PersonNegotiationRole> personsRoles) {
     return personsRoles.stream()
-        .map(
-            personRole ->
-                new PersonRoleDTO(
-                    personRole.getPerson().getAuthName(), personRole.getRole().getName()))
-        .collect(Collectors.toSet());
+            .map(
+                    personRole ->
+                            new PersonRoleDTO(
+                                    personRole.getPerson().getId().toString(),
```

Comment:
String.valueOf(personRole.getPerson().getId()) should be better in orde to avoid a NullPointerException (even if it's not possible for the Id to be Null)